lib/webauto/web4cucumber.rb

```diff
@@ -80,7 +80,9 @@ def browser
       if ENV.has_key?("http_proxy") || @http_proxy
         # get rid of the heading "http://" that breaks the profile
         proxy_raw = @http_proxy || ENV["http_proxy"]
+        logger.error "debug proxy_raw: #{proxy_raw}"
```

Comment:
Logging an error may lead to other team members thought there are something wrong, and report issue to our team.
To avoid that, I would suggest use `logger.warn`, or update the message to `For UI team to debug proxy_raw blablabal`